index.html

```diff
@@ -15,15 +15,14 @@
 		<meta property="og:image" content="https://raw.github.com/tastejs/todomvc/gh-pages/site-assets/screenshot.png">
 		<meta property="og:description" content="Helping you select an MV* framework - Todo apps for Backbone.js, Ember.js, AngularJS, Spine and many more">
 		<link rel="shortcut icon" href="site-assets/favicon.ico">
-		<script src="bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
 		<!-- build:remove -->
 		<script src="bower_components/prefixfree/prefixfree.min.js"></script>
 		<!-- endbuild -->
 		<!-- build:css site-assets/main.min.css -->
 		<link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
-		<link rel="stylesheet" href="site-assets/main.css">
+		<link rel="stylesheet" href="site-assets/main.min.css">
 		<!-- endbuild -->
-		<link rel="import" href="bower_components/paper-tabs/paper-tabs.html">
+		<link rel="import" href="bower_components/paper-tabs/paper-tabs.html" async>
```

Comment:
It doesn't look like the FOUC case is being handled here. @matheuspoleza thoughts?